### PR TITLE
Fix Radix Param, nested ternary, expected '===', console.log

### PR DESCRIPTION
**Pull request by abidr**
#34 Issue
- Fixed Radix parameters error
- Changed nested ternary to regular expression
- changed == to === for type check
- Ignored console.log (it might be needed)

### CODE REVIEW

File: server/src/services/product-service.js [hunk 1]
```diff
@@ -54,20 +54,25 @@ const search = async (payload) => {
...
+  let query;
+
+  if (queries.length > 1) {
+    query = { $and: queries };
+  } else if (queries.length === 1) {
+    const queryFirst = queries[0];
+    query = queryFirst;
+  } else {
+    query = {};
+  }
```
Comment (foyzulkarim) [inline server/src/services/product-service.js:66]: How about this?
```
let query = {};
if(queries.length === 1) {
    // code
}
if(queries.lengh > 1){
    // code 
}

```

File: server/src/services/product-service.js [hunk 2]
```diff
@@ -105,27 +110,35 @@ const count = async (payload) => {
...
```
Comment (foyzulkarim) [inline server/src/services/product-service.js:122]: same suggestion as above.

**Review by foyzulkarim — CHANGES_REQUESTED** [review]
Thanks brother. I added some suggestion. Please have a look. :)

**Comment by foyzulkarim** [comment]
@abidr brother, do you want to work on this anytime soon?

**Comment by abidr** [comment]
> @abidr brother, do you want to work on this anytime soon?

Yes I'll make the changes today.

**Comment by foyzulkarim** [comment]
this pull request is obsolete now. closing it. thanks for your efforts

**Pull request closed without merging**